api/src/database/migrations/20210510A-restructure-relations.ts

```diff
@@ -12,7 +12,7 @@ export async function up(knex: Knex): Promise<void> {
 	await knex('directus_relations').update({ one_deselect_action: 'nullify' });
 
 	await helper.changeToType('directus_relations', 'sort_field', 'string', {
-		length: 64,
+		length: helper.getColumnNameMaxLength(),
```

Comment:
In the name of consistency should we use `helper.schema. getColumnNameMaxLength` and update usage accordingly?

Reply:
Resolved in 3c90df309ef0013c85375db421230ad84187be09